### DISCUSSION TITLE
Can Appwrite serve for 1M users

### DISCUSSION THREAD

**Discussion by nguyenthanhtien** [appwrite/appwrite#7196] [Q&A]
Hello, i want ask about AppWrite, our team consider using AppWrite for production, i want ask if using AppWrite, 1M users concurrent, it can serve ok or not? Can Appwrite support another database with Index/Function/Store Procedure features like Postgresql, SQL Server?

**Comment by SmollaL1st** [comment]
It all depends on the power of your server. In fact, this is possible, but no one has checked it, or maybe they have, I personally don’t know.

**Reply by minecrawler** [reply]
I think the question is more geared towards Appwrite Cloud. At the moment, it struggles with 1 user, depending on the time. I really hope to see better stats once paid tiers are introduced, else we'll have to get our own server, which is inconvenient for many reasons.
